java/src/main/java/org/apache/zeppelin/java/JavaInterpreter.java

```diff
@@ -17,19 +17,20 @@
 
 package org.apache.zeppelin.java;
 
+import java.io.File;
+import java.util.Collections;
+import java.util.List;
+import java.util.Properties;
+import java.util.UUID;
+import java.util.stream.Stream;
```

Comment:
this is Java 8 + only?

Reply:
Yes `@since 1.8`.
But after [ZEPPELIN-621](https://jira.apache.org/jira/browse/ZEPPELIN-621), is this not a problem?